src/config.h

```diff
@@ -0,0 +1,244 @@
+/* config.h.  Generated from config.h.in by configure.  */
```

Comment:
This file depends on the user environment, so it should be generated by the build system, and not be included in the library.

Reply:
Yeah I know. This was just a hack I did to get it working on my system. 

Generating this file from the user environment is something I was planning on implementing with cmake later. 

I was just prioritizing getting it building on my system so I could focus on cif stuff.